docs/docs/noir/concepts/control_flow.md

```diff
@@ -43,3 +28,38 @@ if a == 0 {
 }
 assert(x == 2);
 ```
+
+## Loops
+
+Noir has one kind of loop: the `for` loop. `for` loops allow you to repeat a block of code multiple
+times.
+
+The following block of code between the braces is run 10 times.
+
+```rust
+for i in 0..10 {
+    // do something
+}
+```
+
+The index for loops is of type `u64`.
+
+### Break and Continue
+
+In unconstrained code, `break` and `continue` are also allowed in `for` loops. These are only allowed
+in unconstrained code since normal constrained code requires that Noir knows exactly how many iterations
+a loop may have. `break` and `continue` can be used like so:
+
+```rust
+for i in 0 .. 10 {
+    if i == 2 {
+        continue;
+    }
+    if i == 5 {
+        break;
+    }
+    println(i);
+}
+```
+
+`break` and `continue` cannot currently be used to jump out of more than a single loop at a time.
```

Comment:
While `break` and `continue` are common, I think it would be useful to make their behavior explicit in the docs. Viz., "The `break` statement continues execution after the body of the loop that it's called from and the `continue` statement continues execution from the next iteration of its containing loop, skipping the current iteration."

Reply:
I added some detail here and added a reference to it in the unconstrained docs as well